utils/build.ps1

```diff
@@ -2172,6 +2172,21 @@ function Build-Sanitizers([Hashtable] $Platform) {
     }
 }
 
+function Build-Brotli([Hashtable] $Platform) {
+  Build-CMakeProject `
+    -Src $SourceCache\brotli `
+    -Bin "$BinaryCache\$($Platform.Triple)\brotli" `
+    -InstallTo "$BinaryCache\$($Platform.Triple)\usr" `
```

Comment:
Do we need to install it? Can we not use it from the build tree?

Reply:
I'm following the example of zlib here, and it feels cleaner to rely on build artifacts being in a consistent place for all dependency outputs. It seems all other dependencies already do this, so I think we should be consistent. (Maybe let's do these kind of changes in a separate PR for all dependencies if you feel strongly about it)

Reply:
Yes, except, the idea with `Brotli_DIR` is that you tell CMake the directory to look into for `BrotliConfig.cmake` and it will wire up the headers and libraries appropriately. You don't need to pass flags or paths to the libraries or headers.

Reply:
Thanks. I will experiment and report back.